lib/Db/FaceMapper.php

```diff
@@ -131,7 +131,7 @@ public function getOldestCreatedFaceWithoutPerson(string $userId, int $model) {
 
 	public function getFaces(string $userId, $model): array {
 		$qb = $this->db->getQueryBuilder();
-		$qb->select('f.id', 'f.person', 'f.confidence', 'f.descriptor')
+		$qb->select('f.id', 'f.person', 'f.bottom', 'f.top', 'f.confidence', 'f.descriptor')
```

Comment:
TODO: We could create two methods, one for the clustering stage, and another for the fronted code to optimize...